src/service/translate.js

```diff
@@ -709,6 +710,29 @@ function $translate($STORAGE_KEY, $windowProvider, $translateSanitizationProvide
 
   /**
    * @ngdoc function
+   * @name pascalprecht.translate.$translateProvider#useForceAsyncReload
+   * @methodOf pascalprecht.translate.$translateProvider
+   *
+   * @description
+   * If force async reload is enabled, async loader will be always called
+   * even if $translationTable already contains the language key.
+   *
```

Comment:
I've tested on our application myself and when activated it concatenated the table using `angular.extend()`. We have both translations provided using `$translationProvider.translations` and `$translateProvider.useStaticFilesLoader`.